scripts/_helpers.py

```diff
@@ -22,6 +23,48 @@
 
 REGION_COLS = ["geometry", "name", "x", "y", "country"]
 
+logger = logging.getLogger(__name__)
+
+
+def handle_exception(exc_type, exc_value, exc_traceback):
+    """
+    Customise errors traceback.
+    """
+
```

Comment:
Please, drop the empty line for PEP8 standard

Reply:
Fixed.